src/App.js

```diff
@@ -14,7 +14,7 @@ import './App.css';
 function App() {
   const boxCard = useRef({});
   const textInput = useRef({});
-  const [password, setPassword] = useState([]);
+  const [password, setPassword] = useState('');
```

Comment:
oh thanks, at first I want to fetch several passwords, but afterall only one selected

Reply:
Sure! At first I was a bit unsure which one is correct, but since `res[0]` in `reGenerate` is a string then I thought a string is the correct one.